packages/react-native/android/src/main/java/io/invertase/notifee/NotifeeReactUtils.java

```diff
@@ -210,12 +210,9 @@ static boolean isAppInForeground() {
     for (ActivityManager.RunningAppProcessInfo appProcess : appProcesses) {
       if (appProcess.importance == ActivityManager.RunningAppProcessInfo.IMPORTANCE_FOREGROUND
           && appProcess.processName.equals(packageName)) {
-        ReactContext reactContext;
-
-        try {
-          reactContext = (ReactContext) context;
-        } catch (ClassCastException exception) {
-          return true;
+        ReactContext reactContext = getReactContext();
+        if (reactContext == null) {
+          return false;
         }
```

Comment:
Not sure what the purpose of this code was, but it looks like it was always failing with `ClassCastException`

Reply:
mmm, this is a part of code I've not dealt with much. What you've done with using `getReactContext()` makes sense